lib/widgets/passcode.dart

```diff
@@ -64,39 +64,47 @@ class LockWallet extends StatelessWidget {
   Widget build(BuildContext context) {
     AuthBloc bloc = Provider.of(context);
 
-    return buildPasscodeScreen(
-      title: "Choose Passcode".i18n,
-      shouldTriggerVerification: _verificationNotifier.stream,
-      passwordEnteredCallback: (passcode) {
-        _verificationNotifier.add(true);
-        SettingsNotifier.of(context).savePasscode(passcode);
-      },
-      isValidCallback: () {},
-      cancelCallback: () {},
-      bottomWidget: Padding(
-        padding: const EdgeInsets.only(top: 32),
-        child: MaterialButton(
-          child: Container(
-            padding: const EdgeInsets.only(left: 17, right: 17, top: 12, bottom: 12),
-            decoration: BoxDecoration(
-              borderRadius: BorderRadius.circular(25),
-              border: Border.all(
-                color: Colors.white,
-              ),
-            ),
-            child: Text(
-              "Disable Passcode".i18n,
-              textAlign: TextAlign.center,
-              style: TextStyle(
-                fontSize: 14,
-                color: Colors.white,
-                fontWeight: FontWeight.w300
+    int disableBoxHeight = 50;
+
+    return SingleChildScrollView(
```

Comment:
Entering passcode should not be a scroll view - it should shrink the entry pad so it fits?!

Reply:
Plz make it shrink instead.

Reply:
This is a library we are using. We dont have control over the pad. 

And on smaller devices is not looking good. We could remove some padding that we added in the initial implementation and that should make it work for smaller screens, but it wont look as nice in larger devices.